### ISSUE TITLE
error: unpack requires a string argument of length 16

### ISSUE THREAD

**Issue by panoptikum** [pdfminer/pdfminer.six#200]
Hello everybody,

First thanks for the great work you put into pdfminer.six. I really appreciate it.

I ran into a bug for which there is already a solution proposed on the site of euske/pdfminer:

``` 
File "/usr/local/lib/python2.7/dist-packages/pdfminer/pdfinterp.py", line 832, in process_page
    self.render_contents(page.resources, page.contents, ctm=ctm)
  File "/usr/local/lib/python2.7/dist-packages/pdfminer/pdfinterp.py", line 843, in render_contents
    self.init_resources(resources)
  File "/usr/local/lib/python2.7/dist-packages/pdfminer/pdfinterp.py", line 347, in init_resources
    self.fontmap[fontid] = self.rsrcmgr.get_font(objid, spec)
  File "/usr/local/lib/python2.7/dist-packages/pdfminer/pdfinterp.py", line 195, in get_font
    font = self.get_font(None, subspec)
  File "/usr/local/lib/python2.7/dist-packages/pdfminer/pdfinterp.py", line 186, in get_font
    font = PDFCIDFont(self, spec)
  File "/usr/local/lib/python2.7/dist-packages/pdfminer/pdffont.py", line 654, in __init__
    StringIO(self.fontfile.get_data()))
  File "/usr/local/lib/python2.7/dist-packages/pdfminer/pdffont.py", line 375, in __init__
    (name, tsum, offset, length) = struct.unpack('>4sLLL', fp.read(16))
error: unpack requires a string argument of length 16
```

bug: https://github.com/euske/pdfminer/issues/144
proposed solution: https://github.com/euske/pdfminer/pull/159

Is this a possible bug in pdfminer.six?

If yes, i might modify the respective files for pdfminer.six as well.

Best

**Comment by pietermarsman** [comment]
Hi @panoptikum,

By any chance, do you still have the code and pdf that shows this bug? I am going to work on #193 but another testing case is good to have.

**Comment by pietermarsman** [comment]
The pdf in #193 does not cause an error any more. What about yours?

**Comment by pietermarsman** [comment]
I'm closing this one since it is fixed for #193 and https://github.com/euske/pdfminer/issues/96 (referenced in  https://github.com/euske/pdfminer/pull/159).
